src/app.py

```diff
@@ -0,0 +1,107 @@
+from flask import Flask
+from flask import redirect
+from flask import render_template
+from flask import request
+import psycopg2
+
+app = Flask(__name__)
+
+conn = psycopg2.connect(
+    host="localhost",
+    database="postgres",
+    user="postgres",
+    password="123")
+
+cursor = conn.cursor()
+
+
+def select_all():
+    cursor.execute("SELECT * from products ORDER BY col_name")
+    rows = cursor.fetchall()
+    return rows
+
+
+def select_something(iden):
+    cursor.execute("SELECT * from products WHERE id=(%s)", (iden,))
+    rows = cursor.fetchall()
+    return rows[0]
+
+
+def delete_something(iden):
+    cursor.execute("DELETE from products WHERE id=(%s)", (iden,))
+    conn.commit()
+
+
+def add_position(name, price, quantity, comment):
+    cursor.execute('''INSERT INTO products
+                   (col_name, col_price, col_quantity, col_comment)
+                   VALUES (%s,%s,%s,%s)''', (name, price, quantity, comment))
+    conn.commit()
+    return "Позиция успешно добавлена"
+
+
+# сделать потом красиво
+def change_some_position(name, price, quantity, comment, iden):
```

Comment:
Добавь везде тайпинги
```suggestion
def change_some_position(name, price, quantity, comment, product_id: int) -> None:
```